doc/material-getting-started.md

```diff
@@ -0,0 +1,113 @@
+# Uno Toolkit Material Library
+<p align="center">
+  <img src="assets/material-toolkit-design-system.png">
+</p>
+
+The Uno.Toolkit.Material library is available as NuGet packages that can be added to any new or existing Uno solution.
+
+## `MaterialToolkitTheme`
+
+There is now a new, simplified method for initializing the Material Toolkit resources. It is recommended to use the new `MaterialToolkitTheme` going forward since `MaterialToolkitResources` has been deprecated.
+
+### Inheritance
+ResourceDictionary &#8594; MaterialToolkitTheme
+
+### Constructors
+| Constructor    | Description                                           |
+|----------------|-------------------------------------------------------|
+| MaterialToolkitTheme()         | Initializes a new instance of the `MaterialToolkitTheme` `ResourceDictionary`.       |
+| MaterialToolkitTheme(`ResourceDictionary?` colorOverride, `ResourceDictionary?` fontOverride)         | Initializes a new instance of the `MaterialToolkitTheme` `ResourceDictionary` and applies the given overrides       |
```

Comment:
dont need to syntax highlight everything; too much is no different than no highlight
```suggestion
| MaterialToolkitTheme() | Initializes a new instance of the `MaterialToolkitTheme` resource dictionary. |
| MaterialToolkitTheme(ResourceDictionary? colorOverride, ResourceDictionary? fontOverride) | Initializes a new instance of the `MaterialToolkitTheme` resource dictionary` and applies the given overrides. |
```